docs/logging-and-managing-experiment-results/downloading-experiment-data.rst

```diff
@@ -0,0 +1,307 @@
+    my_exp.download_artifacts('model_checkpoints/')
+
+Source code
+"""""""""""
+Download source code used un the experiment as a ZIP archive.
+
+.. code-block:: python3
+
+    # Download all sources to the cwd
+    my_exp.download_sources()
+
+.. note::
+
+    You can also download source directly from the UI: :ref:`here is how <download-from-neptune-ui>`.
+
+More options
+""""""""""""
+Besides metrics, artifacts and scripts covered above, you can use other methods as well. Here is a full list of methods that download data:
+
+* :meth:`~neptune.experiments.Experiment.get_hardware_utilization`: Gets GPU, CPU and memory utilization data.
+* :meth:`~neptune.experiments.Experiment.get_logs`: Gets all log names with their most recent values for this experiment.
+* :meth:`~neptune.experiments.Experiment.get_numeric_channels_values`: Gets values of specified metrics (numeric logs).
+* :meth:`~neptune.experiments.Experiment.get_parameters`: Gets parameters for this experiment.
+* :meth:`~neptune.experiments.Experiment.get_properties`: Gets user-defined properties for this experiment.
+* :meth:`~neptune.experiments.Experiment.get_system_properties`: Gets experiment properties.
+* :meth:`~neptune.experiments.Experiment.get_tags`: Gets the tags associated with this experiment.
+* :meth:`~neptune.experiments.Experiment.download_artifact`: Download an artifact (file) from the experiment storage.
+* :meth:`~neptune.experiments.Experiment.download_artifacts`: Download a directory or a single file from experiment’s artifacts as a ZIP archive.
+* :meth:`~neptune.experiments.Experiment.download_sources`: Download a directory or a single file from experiment’s sources as a ZIP archive.
+* :meth:`~neptunecontrib.api.utils.get_pickle`: Download pickled artifact (file) from Neptune and returns a Python object.
+
+Combining downloading methods
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+You can combine few downloading options to build custom visualizations or analysis. Example below shows how to use :meth:`~neptune.projects.Project.get_experiments` and :meth:`~neptune.experiments.Experiment.get_numeric_channels_values` and seaborn library to overlay metric from multiple experiments on the same plot.
+
+Get list of :class:`~neptune.experiments.Experiment` objects.
+
+.. code-block:: python3
+
+    import neptune
+
+    # Set project
+    project = neptune.init('my_workspace/my_project')
+
+    # Get list of experiments
+    experiments = project.get_experiments(owner='...', tag='...')
+
+Download metrics data from all experiments in the list, by using :meth:`~neptune.experiments.Experiment.get_numeric_channels_values`
+
+.. code-block:: python3
+
+    for experiment in experiments:
+        raw_df = experiment.get_numeric_channels_values(*metrics_names)
+        metrics_df = raw_df[...]
+
+Prepare dataframe with top performing experiments and make seaborn plot
+
+.. code-block:: python3
+
+    # Prepare dataframe
+    top_acc_df = metrics_df.sort_values(...)
```

Comment:
if we say top_acc then in the sort_values we have those acc right?

Reply:
changed